oslo/transformers/oslo_init.py

```diff
@@ -0,0 +1,451 @@
+import json
+import logging
+from copy import deepcopy
+import torch
+from typing import List, Tuple
+from dataclasses import dataclass
+from oslo.torch.distributed import ParallelContext
+from oslo.torch.nn.parallel import (
+    PipelineParallel,
+    TensorParallel,
+)
+from oslo.torch.nn.parallel.sequence_parallel import SequenceParallel
+from oslo.torch.nn.parallel.data_parallel.data_parallel import DataParallel
+from oslo.torch.nn.parallel.data_parallel._ddp.distributed_data_parallel import (
+    DistributedDataParallel,
+)
+
+from oslo.torch.distributed.parallel_mode import ParallelMode
+from .trainer_utils import (
+    log_dist
+)
+
+NoneType = type(None)
+
+
+def _type(_dtype):
+    return lambda key, val: {
+        "check": isinstance(val, _dtype),
+        "msg": f"{key}: {val} is not a valid set. it must be type of {_dtype}",
+    }
+
+
+def _values(*args):
+    return lambda key, val: {
+        "check":
+            val in args,
+        "msg":
+            f"{key}: {val} is not a valid set. it must be one of {list(args)}",
+    }
+
+
+def _dp_config(*args):
+    params = {}
+    for arg in args:
+        params.update(DATA_PARALLEL_CONFIGS[arg])
+    return params
+
+
+DATA_PARALLEL_CONFIGS = {
```

Comment:
We can remove all of these. 
Please check new DataParallel API !

Reply:
Ok. DataParallel config in oslo_init would be removed